pkgs/tools/typesetting/lowdown/default.nix

```diff
@@ -24,12 +24,20 @@ stdenv.mkDerivation rec {
   '';
 
   # Fix lib extension so that fixDarwinDylibNames detects it
-  postInstall = lib.optionalString stdenv.isDarwin ''
+  postInstall = lib.optionalString (stdenv.isDarwin && !stdenv.isAarch64) ''
```

Comment:
Just curious: Is aarch64-darwin somehow different here or does fixDarwinDylibNames work differerntly?

Reply:
This probably isn't needed when I think about it.
aarch64 is different in few ways (like needs sigtool and calls launchd's not in x86 to name a few)

Reply:
wasn't needed, so it's removed